src/NuGet.Server/Infrastructure/ServerPackage.cs

```diff
@@ -79,8 +39,10 @@ public ServerPackage(IPackage package, PackageDerivedData packageDerivedData)
             MinClientVersion = package.MinClientVersion;
             ReportAbuseUrl = package.ReportAbuseUrl;
             DownloadCount = package.DownloadCount;
-            IsAbsoluteLatestVersion = package.IsAbsoluteLatestVersion;
-            IsLatestVersion = package.IsLatestVersion;
+            SemVer1IsAbsoluteLatest = false;
+            SemVer1IsLatest = false;
+            SemVer2IsAbsoluteLatest = false;
+            SemVer2IsLatest = false;
```

Comment:
We could make a generalized mapping from SemVerLevel/ClientCompatibility to `IsLatest`/`IsAbsoluteLatest` but I think it would overcomplicate things at this point.